lib/matplotlib/_pylab_helpers.py

```diff
@@ -84,6 +84,7 @@ def destroy_fig(cls, fig):
 
     @classmethod
     def destroy_all(cls):
+        import gc
```

Comment:
I suppose it would also be worth adding a comment that this is here deliberately (since `gc` is already imported at the module level any delinter is going to complain about this...)